internal/pkg/cli/secret_init.go

```diff
@@ -305,9 +410,16 @@ func BuildSecretInitCmd() *cobra.Command {
 			if err := opts.Ask(); err != nil {
 				return err
 			}
-			if err := opts.Execute(); err != nil {
+
+			err = opts.Execute()
+			if opts.shouldShowOverwriteHint {
+				log.Warningf("If you want to overwrite an existing secret, use the %s flag.\n", color.HighlightCode(fmt.Sprintf("--%s", overwriteFlag)))
```

Comment:
is this duplicated to https://github.com/aws/copilot-cli/pull/2301/files#diff-f89e17f1e021d27071747e80e78111d9fb921a1d3af52a0346b0de25baef4315R356 ?

Reply:
addressed!